app/scripts/models/app-settings-model.js

```diff
@@ -33,6 +33,7 @@ const AppSettingsModel = Backbone.Model.extend({
         fontSize: 0,
         tableViewColumns: null,
         generatorPresets: null,
+        generatorHide: false,
```

Comment:
let's rename it to `generatorHidePassword` to make it more obvious

Reply:
Okay, makes sense.

Reply:
d96f458